app/controllers/tt/api/v1/split_registries_controller.rb

```diff
@@ -1,5 +0,0 @@
-class Tt::Api::V1::SplitRegistriesController < Tt::Api::V1::ApplicationController
-  def show
-    @active_splits = TestTrack::FakeServer.split_registry
-  end
-end
```

Comment:
Do we wanna keep this for local dev including the js client until the js client is updated?

Reply:
Yeah probs eh?

Reply:
Turns out this was actually still conforming to the v1 interface :-/ I gotta fix it and support both. Though honestly, I'm not sure this endpoint is ever used by the JS client - the JS client only becomes aware of the TT server's URL from the split payload embedded in the page, with the split registry. Maybe I can kill them altogether instead. Will look into it.

Reply:
hmm, good point. the js client always gets the split registry from the page.

Reply:
pulled it out.